### PR TITLE
Google Trends Screensaver

### PR DESCRIPTION
**Pull request by pamo**
Adds cask for screensaver installation package.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/google-trends-screensaver.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'google-trends-screensaver' do
+  version :latest
+  sha256 :no_check
```
Comment (tolkonepiu) [inline Casks/google-trends-screensaver.rb]: 👎

File: Casks/google-trends-screensaver.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'google-trends-screensaver' do
```
Comment (vitorgalvao) [inline Casks/google-trends-screensaver.rb]: Call it just `google-trends`.

File: Casks/google-trends-screensaver.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/google-trends-screensaver.rb]: @tolkonepiu Unless you have an official versioned URL we can use, that’s the way it needs to be.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**